optimum/bettertransformer/models/base.py

```diff
@@ -61,6 +63,13 @@ def __init__(self, config):
                 self.num_layers = getattr(config, attr)
                 break
 
+        if old_layer is not None:
```

Comment:
I think we should refer the pre-transformation layer as "original layer" instead of "old layer".
```suggestion
        if orig_layer is not None:
```

Reply:
fixed in [8ee8df5](https://github.com/huggingface/optimum/pull/676/commits/8ee8df53d6685110ca2db78b94e9b0c0f49d533e)